app/src/main/java/ch/epfl/favo/common/DatabaseWrapper.java

```diff
@@ -25,7 +24,7 @@
 
   // final fields regarding ID generation
   private static final String ID_CHARS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
-  private static final int ID_LENGTH = 25;
+  private static final int ID_LENGTH = 28;
```

Comment:
rather than using the generateRandomId() we can use 
UUID uuid = UUID.randomUUID(); and change the ID type to UUID